### DISCUSSION TITLE
怎么设置背景颜色？

### DISCUSSION THREAD

**Discussion by guanpj** [oldwinter/knowledge-garden#60] [提问] [answered]
根据你的指引部署后背景是纯白的，但看你的样式有个背景，觉得挺好看的，请问应该怎么设置？

**Comment by oldwinter** [comment, marked as answer]
抱歉我没研究过，直接fork的原作者的样式的。应该是style文件夹下的scss样式文件控制的。如果有前端基础，你可以自己打开控制台看一下是哪个class控制背景色自己改一下。 tips，样式代码直接问chatgpt有惊喜。

**Reply by guanpj** [reply]
嗯，我看对比了一下，确实有差别。感谢回复！

**Reply by guanpj** [reply]
另外，补充一下：要达到你站点的效果，最好还是直接 fork 你的库。这个朋友写得很好：https://ob.tianzhongs.ml/001_%E9%83%A8%E7%BD%B2Obsidian%E9%9D%99%E6%80%81%E7%9F%A5%E8%AF%86%E5%BA%93%E7%BD%91%E7%AB%99
看能不能找个地方说明一下。

**Comment by oldwinter** [comment]
赞，谢谢推荐。这个文章总结得很漂亮。我要收纳文章和网站，造福更多人。

在 2023年3月21日星期二，guanpj ***@***.***> 写道：
